sklearn/model_selection/tests/test_validation.py

```diff
@@ -936,6 +937,124 @@ def test_cross_val_predict_with_method():
         assert_array_almost_equal(expected_predictions, predictions)
 
 
+def test_cross_val_predict_corner_case():
```

Comment:
No. There's not much benefit in extreme brevity for test names. On the other hand "edge case" is highly undescriptive of which edge case when there are no doubt many.